src/Client.js

```diff
@@ -610,6 +610,37 @@ class Client extends EventEmitter {
         return ContactFactory.create(this, contact);
     }
 
+    async getMessageById(messageId) {
+        const msg = await this.pupPage.evaluate(async messageId => {
+            let msg = window.Store.Msg.get(messageId);
+            if(msg) return window.WWebJS.getMessageModel(msg);
+
+            const params = messageId.split('_');
+            if(params.length !== 3) throw new Error('Invalid serialized message id specified');
```

Comment:
We need to check param.lenght before doing Msg.get (To avoid errors)

Reply:
good call